components/integrations/edit_oauth_app/edit_oauth_app.test.tsx

```diff
@@ -21,9 +26,22 @@ describe('components/integrations/EditOAuthApp', () => {
         update_at: 1501365458934,
         callback_urls: ['https://test.com/callback', 'https://test.com/callback2'],
     };
-    const team = {
+    const team: Team = {
```

Comment:
Instead of specifying all these extra fields, you could either use `TestHelper.getTeamMock` from `utils/test_helper` or if those fields aren't needed, you can use `const team = {...} as Team;` to force the type to be treated as `Team` despite the missing fields. Either way, you could save some typing that way.